model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -156,3 +156,8 @@ internal data class OrtConfigurationWrapper(
  * A typealias for key-value pairs, used in several configuration classes.
  */
 typealias Options = Map<String, String>
+
+/**
+ * The filename of the reference configuration file.
+ */
+const val REFERENCE_CONFIG_FILENAME = "reference.conf"
```

Comment:
We have `ORT_CONFIG_FILENAME` in `Constants.kt`. So should we put `REFERENCE_CONFIG_FILENAME` there, too? Or move `ORT_CONFIG_FILENAME` to here?

Reply:
I thought about it, but this constant references a file which is part of the model module while the other constants refer to user files, so it made more sense to me to keep it here.

Reply:
I'd still argue it belongs close to `const val ORT_CONFIG_FILENAME = "ort.conf"`, but I won't insist on it.